src/components/StateMeta.js

```diff
@@ -59,6 +59,24 @@ function StateMeta({stateCode, data, timeseries}) {
   const growthRate =
     (Math.pow(lastConfirmed / prevWeekConfirmed, 1 / 7) - 1) * 100;
 
+  const percentageDescription = (percentageValue, description) => {
+    let confirmedCasesText 
+
+    if (percentageValue > 0 && percentageValue < 1)
```

Comment:
A small inconsistency here 
* Since `percentageDescription` is always called for percentageValue > 0 , no need to check for `percentageValue > 0`
* Its good to test `percentageValue > 0` but a case when percentage value <= 0 has to be handled then